scripts/nginx/autobrr.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# autobrr nginx conf
+# ludviglundgren 2021 for Swizzin
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+users=($(_get_user_list))
+
+if [[ ! -f /etc/nginx/apps/autobrr.conf ]]; then
+
+    cat > /etc/nginx/apps/autobrr.conf << 'AUTOBRR'
+location /brr {
```

Comment:
lol :p are you sure `/brr` 301ing to `/autobrr/` won't be a problem with double-slashes? I never know myself about this when it comes to these nginx things

Reply:
Maybe not. Removed it 👍